src/components/Card/Card.tsx

```diff
@@ -9,9 +9,9 @@ interface CardProps {
 const Card = ({ title, image }: CardProps): JSX.Element => {
   return (
     <div className="card-container">
-      <article className="news-card">
-        <img className="article-image" src={ image } alt={ title } />
-        <h2 className="article-title">{ title }</h2>
+      <article className="news-card cy-article-card">
+        <img className="article-image cy-article-image" src={ image } alt={ title } />
+        <h2 className="article-title cy-article-title">{ title }</h2>
```

Comment:
Is this a "best practice" I don't know about?! The "cy" in front of the className? I'd love to more about it.

Reply:
Yep, it's a best practice. See the Cypress [docs](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)